shell/circleci/machine.sh

```diff
@@ -47,24 +47,12 @@ if ! command -v gh >/dev/null; then
   rm ./gh.deb
 fi
 
-echo "Installing yq"
+echo "Removing yq"
 # Remove the existing yq, if it already exists
 # (usually the Go Version we don't support)
+# We can use gojq via a yq wrapper which uses gobin.sh
 sudo rm -f "$(command -v yq)"
```

Comment:
**thought:** Hah, YOLO nuke `yq`. I like it.

You probably don't need to nuke whatever exists given your wrapper, though.

Reply:
Yeah upon reflection, I don't think we can actually do this change right now since there's a bunch of other scripts which depend on `yq` at the moment. Maybe later if I can force install a `yq` wrapper.